examples/lib/rbac.rego

```diff
@@ -0,0 +1,21 @@
+package lib.rbac
+
+import data.lib.core
+
+rule_has_verb(rule, verb) {
+    verbs := ["*", lower(verb)]
+    verbs[_] == lower(rule.verbs[_])
+}
+
+rule_has_resource_type(rule, type) {
+    types := ["resourceall", lower(type)]
```

Comment:
question: Is this a valid entry for resources? It looks like `ResourceAll` is the name given to the enum var in the API, but I think this would be the same as `verb` and would be `*`.

Reply:
You are absolutely correct! I was a little too literal when reading the K8s documentation earlier today 😄